src/main/default/lwc/lookup/lookup.js

```diff
@@ -14,9 +13,12 @@ export default class Lookup extends LightningElement {
     @api isMultiEntry = false;
     @api hideResults = false;
     get showResults() {return this.isMultiEntry && !this.hideResults}
-    @api errors = [];
     @api scrollAfterNItems;
     @api iconResources = {};
+    @api errors;
+    @api hasError;
```

Comment:
I think this is unnecessary. The same can be achieved with a getter:

get hasErrors() {
    return this.errors.length !== 0;
}